docs/getting-started.md

```diff
@@ -180,3 +191,44 @@ $ molecule test
     only. It is useful if you want to test one platform docker image.
 
 [galaxy development guide]: https://docs.ansible.com/ansible/latest/galaxy/dev_guide.html
+
+## Test the collection playbook or role
+
+So you now have an isolated test environment, and can also use it for live development, by running `molecule converge`.
+It will run through the same steps as above but will stop after the converge action.
+Then you can make changes to your collection or the Converge play, and then run molecule converge again (and again) until you're done with your development work.
+
+One of the default files created as part of the initialization is the `converge.yml` file.
+This file is a playbook created to run your role from start to finish.
+This can be modified if needed but is a good place to start if you have never used Molecule before.
+
+To test the playbook from a collection,
+let's add a task to our role under `tasks/main.yml` file like so:
+
+```
+---
+- name: Include a playbook from a collection
+  ansible.builtin.import_playbook: foo.bar.my_playbook
+
+```
```

Comment:
At this point we haven't edited the default `converge.yml`  - is there a step missing or will this work with whatever was created by the init command?